README.md

```diff
@@ -1,7 +1,18 @@
-`openzipkin/jre-full` is a minimal but full JRE Docker image based on [openjdk:8-jre-alpine](https://github.com/docker-library/openjdk/blob/master/8/jdk/alpine/Dockerfile). 
+`openzipkin/jre-full` is a minimal but full Alpine JRE Docker 1.8 image.
 
 On Dockerhub: [openzipkin/jre-full](https://hub.docker.com/r/openzipkin/jre-full/)
 
 ## Release process
 
-New versions are built on [Travis CI](https://travis-ci.org/openzipkin/docker-jre-full). To trigger a build, push a new tag to GitHub. The tag will be the Docker tag assigned to the newly built image. Name the tag according to the JDK in use. For example `1.8.0_112`.
\ No newline at end of file
+New versions are built on [Travis CI](https://travis-ci.org/openzipkin/docker-jre-full). To trigger a build, push a new tag to GitHub. The tag will be the Docker tag assigned to the newly built image. Name the tag according to the JDK in use.
+
+The tag will be the Docker tag assigned to the newly built image. Name the tag according to the JDK variant use.
+
+For example, with the following output from `docker run (image built) -version`:
+```
+openjdk version "1.8.0_252"
+OpenJDK Runtime Environment (IcedTea 3.16.0) (Alpine 8.252.09-r0)
+OpenJDK 64-Bit Server VM (build 25.252-b09, mixed mode)
+```
+
+You would name the tag `1.8.0_252`.
```

Comment:
this is literally the current version published to alpine edge at the moment. openjdk abandoned at 212 but they only repackaged alpine doing the same